src/fp-ts/createNonEmptyArrayFromArray.ts

```diff
@@ -0,0 +1,15 @@
+import * as t from 'io-ts'
+import { NonEmptyArray, fromArray } from 'fp-ts/lib/NonEmptyArray'
+
+const getNonEmptyArrayFromArray = <A = never>() =>
+  new t.Type<NonEmptyArray<A>, Array<A>, Array<A>>(
+    'NonEmptyArrayFromArray',
+    (m): m is NonEmptyArray<A> => m instanceof NonEmptyArray,
+    (as, c) => fromArray(as).foldL(() => t.failure(as, c), t.success),
+    nep => nep.toArray()
+  )
+
+export const createNonEmptyArrayFromArray = <A, O = A>(
```

Comment:
Please rename `createNonEmptyArrayFromArray` to `nonEmptyArray`, is shorter and more aligned to the existing `array` combinator

Reply:
I wonder, is the default `O = A` meaningful in this case?

Reply:
> Please rename createNonEmptyArrayFromArray to nonEmptyArray, is shorter and more aligned to the existing array combinator

Never mind, I noticed that you are following the convention we adopted so far `createOptionFromNullable`, `createEitherFromJSON`, etc..

Reply:
sorry for late reply.
Yes I'm following the (verbose IMHO) convention.
I would be also in favour of shorter names.
Maybe that's something we can schedule with a proper deprecation process?